google-cloud-trace/google-cloud-trace.gemspec

```diff
@@ -0,0 +1,27 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-trace"
+  gem.version       = "0.21.0"
+
+  gem.authors       = ["Google Inc"]
+  gem.email         = ["googleapis-packages@google.com"]
+  gem.description   = "a grpc-based api"
```

Comment:
The field in the existing error reporting and monitoring gems should be fixed as well.